code/game/objects/noose.dm

```diff
@@ -0,0 +1,153 @@
+/obj/item/stack/cable_coil/building_checks(datum/stack_recipe/R, multiplier)
+	if(R.title == "noose")
+		if(!(locate(/obj/structure/chair) in usr.loc))
+			to_chat(usr, "<span class='warning'>You have to be standing on top of a chair to make a noose!</span>")
+			return FALSE
+	return ..()
+
+/obj/structure/chair/noose //It's a "chair".
+	name = "noose"
+	desc = "Well this just got a whole lot more morbid."
+	icon_state = "noose"
+	icon = 'icons/obj/objects.dmi'
+	layer = FLY_LAYER
+	flags_1 = NODECONSTRUCT_1
+	var/image/over
```

Comment:
Why is this not a `mutable_appearance`?

Reply:
Which line? I'm still not too familiar with byondcode tbh

Reply:
`var/image/over`

Reply:
not sure, was direct ported from hippie, ill fix that though